circuit/src/ecc/same_curve/mod.rs

```diff
@@ -1058,7 +1219,31 @@ mod test {
 				layouter.namespace(|| "to_sub assigner"),
 			)?;
 
-			Ok((to_add, to_sub))
+			let mut aux_init = to_add.clone();
+			let mut aux_inits: Vec<AssignedPoint<C, NUM_LIMBS, NUM_BITS, P>> = vec![aux_init.clone()];
+			for _ in 1..self.batch_length {
+				aux_inits.push(aux_init.clone());
+				let double_chip = EccDoubleChipset::new(aux_init);
+				aux_init = double_chip.synthesize(
+					common,
+					&config.ecc_double,
+					layouter.namespace(|| "init_double"),
+				)?;
+			}
+
+			let mut aux_fin = to_sub.clone();
+			let mut aux_fins: Vec<AssignedPoint<C, NUM_LIMBS, NUM_BITS, P>> = vec![aux_fin.clone()];
+			for _ in 1..self.batch_length {
+				aux_fins.push(aux_fin.clone());
+				let double_chip = EccDoubleChipset::new(aux_fin);
+				aux_fin = double_chip.synthesize(
+					common,
+					&config.ecc_double,
+					layouter.namespace(|| "fin_double"),
+				)?;
```

Comment:
Adding to array should be below the chip now.
```suggestion
				let double_chip = EccDoubleChipset::new(aux_fin);
				aux_fin = double_chip.synthesize(
					common,
					&config.ecc_double,
					layouter.namespace(|| "fin_double"),
				)?;
				aux_fins.push(aux_fin.clone());
```
Im trying to avoid calling the chip if its not needed. We will move this AuxAssigned chipset outside the tests at some point